node/src/components/consensus/protocols/highway.rs

```diff
@@ -141,13 +141,13 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
 
         let round_success_meter = prev_cp
             .and_then(|cp| cp.as_any().downcast_ref::<HighwayProtocol<I, C>>())
-            .map(|highway_proto| highway_proto.next_era_round_succ_meter(era_start_time))
+            .map(|highway_proto| highway_proto.next_era_round_succ_meter(now))
```

Comment:
Actually, should we make it `max(era_start_time, now)` (possibly even in all the places you've changed)? It might not be relevant in the mainnet anymore, but nodes used to start a significant amount of time before genesis and just using `now` would do weird things to such a situation.

Reply:
Good idea! Will do.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/1356/commits/90edb764bdd9b010405b043f057ed618e704051d.